plugins/modules/packaging/os/pacman.py

```diff
@@ -428,6 +439,9 @@ def remove_packages(self, pkgs):
             self.add_exit_infos("Would have removed %d packages" % len(removed_pkgs))
             return
 
+        if nosave_args:
+            cmd_base.extend(nosave_args)
+
         # actually do it
         cmd = cmd_base + pkg_names_to_remove
```

Comment:
Why not
```suggestion
        cmd = cmd_base + nosave_args + pkg_names_to_remove
```
instead of the above `if`?

Reply:
no reason really, kept the same style as  force_args and extra_args which could also be done like this.

Reply:
does it look better with the new commit?